### PR TITLE
Implemented booking for counter

### PR DESCRIPTION
**Pull request by justin-obrien-smoothstack**
Includes #10 and #11, so after those get merged there will be fewer changes to review here.

Used Stripe Elements to take card info.

This method of using Elements seems to conflict with importing Stripe's JS file and setting up the publishable key in `index.html`. Hence, that part of `index.html` has been commented out, which will break the agent booking as it currently exists.

### CODE REVIEW

File: src/app/counter/counter/counter.component.ts [hunk 1]
```diff
@@ -43,4 +48,9 @@ export class CounterComponent implements OnInit {
     document.getElementById("book").classList.remove("side-link-active");
     document.getElementById("cancel").classList.add("side-link-active");
   }
+
+  resetLinks() {
+    document.getElementById("book").classList.remove("side-link-active");
```
Comment (trevorhuis-smoothstack) [inline src/app/counter/counter/counter.component.ts:53]: Did this solve the issue of the sidebar remaining active?

File: src/app/counter/counter/counter.component.html [hunk 2]
```diff
@@ -13,24 +13,32 @@ <h5><strong>Employee:</strong><br />{{this.counter.name}}</h5>
...
+        routerLink="/counter/traveler"
+        (click)="resetLinks()"
+      >
+        Change
+      </button>
+    </li>
     <li
       id="book"
       class="side-link"
+      routerLink="/counter/booking"
```
Comment (trevorhuis-smoothstack) [inline src/app/counter/counter/counter.component.html:32]: Does the sidebar stay if you use the routerLink?

Comment (trevorhuis-smoothstack) [reply src/app/counter/counter/counter.component.html:32]: I figured this out in the routing file.

File: src/app/counter/counter/counter.component.ts (hunk 1 above)
Comment (justin-obrien-smoothstack) [reply src/app/counter/counter/counter.component.ts:53]: Nope

Comment (justin-obrien-smoothstack) [reply src/app/counter/counter/counter.component.ts:53]: Actually it does solve it when clicking the button to change the traveler, but the issue still occurs when clicking the Counter link in the top nav.

**Pull request merged**